shopfloor_base/utils.py

```diff
@@ -115,3 +115,22 @@ def load_components_without_registry(env, *component_classes):
         return []
     work = WorkContext(model_name=rest_service._name, collection=collection)
     return [comp(work) for comp in component_classes]
+
+
+def purge_endpoints(env, service_usage):
```

Comment:
So once merged, we should write uninstall hook using this helper for all Shopfloor modules?